generated_cpp_50/PythonQt_QtBindings.cpp

```diff
@@ -13,6 +13,7 @@ void PythonQt_init_QtOpenGL(PyObject*);
 void PythonQt_init_QtXml(PyObject*);
 void PythonQt_init_QtXmlPatterns(PyObject*);
 void PythonQt_init_QtUiTools(PyObject*);
+void PythonQt_init_QtMultimedia(PyObject*);
```

Comment:
I don't know why these generated bindings are in the source tree. In my CTK build bindings are generated  during build (in c:\D\S4D\CTK-build\PythonQt-build\generated_cpp_511).